uk.co.compucorp.civicrm.tasksassignments/views/dashboard/tasks.html

```diff
@@ -92,8 +91,15 @@ <h3 class="panel-title">{{list.label[list.filterParams.due]}}</h3>
           </div>
         </div>
         <ul class="{{prefix}}list-task">
-          <li ng-repeat="task in listFiltered = (listUnlimited = (listOngoing = (list.list | filterByStatus:cache.taskStatusResolve:false) | filterByDateField:list.filterParams.due:list.filterParams.dateRange | filterByOwnership:list.filterParams.ownership | filterByAssignmentType:list.filterParams.assignmentType | filterByContactId:list.filterParams.contactId | orderBy: 'activity_date_time') | limitTo: listLimit)"
-            ng-controller="TaskCtrl" ng-class="{ '{{prefix}}task-completed': task.completed, '{{prefix}}task-due': task.due }" ng-include src="'task.html'" ct-spinner ct-spinner-id="task{{task.id}}">
+          <li ng-repeat="task in listFiltered = (listUnlimited = (listOngoing = (list.filterByDateField(list.filterParams.due)
```

Comment:
I would ask to organise this code a bit differently, if you don't mind:

```html
<li ng-repeat="task in listFiltered = 
  (listUnlimited = 
    (listOngoing = 
      (list.list | filterByStatus:cache.taskStatusResolve:false) 
      | filterByDateField:list.filterParams.due:list.filterParams.dateRange 
      | filterByOwnership:list.filterParams.ownership 
      | filterByAssignmentType:list.filterParams.assignmentType 
      | filterByContactId:list.filterParams.contactId 
      | orderBy: 'activity_date_time'
    ) | limitTo: listLimit)"
  ng-controller="TaskCtrl"
  ng-class="{ '{{prefix}}task-completed': task.completed, '{{prefix}}task-due': task.due }"
  ng-include src="'task.html'"
  ct-spinner
  ct-spinner-id="task{{task.id}}">
```

Reply:
Looks better! will do.

Reply:
Could you please indent what's inside ng-repeat by 2 spaces? It looks a bit odd that <li> and (listUnlimited have the same indentation level. Please have a look at my snippet ^^

Reply:
Fixed!